scripts/core/PROG.rst

```diff
@@ -1736,6 +1792,16 @@ The following examples demonstrate proper usage for a system configuration of tw
 - `1.1, 1.2`: only device 1 is reported as device 0; with its sub-devices 1 and 2 reported as sub-devices 0 and 1, respectively
 - `0.2, 1.3, 1.0, 0.3`: both device 0 and 1 are reported; device 0 reports sub-devices 2 and 3 as sub-devices 0 and 1, respectively; device 1 reports sub-devices 0 and 3 as sub-devices 0 and 1, respectively; the order is unchanged.
 
+%if ver >= 1.7:
+The following examples show the use of different values in the ${X}_AFFINITY_MASK when setting ${X}_FLAT_DEVICE_HIERARCHY to 1, in the
+same system with two devices and four sub-devices.
+
+- `0, 1, 2, 4`: all sub-devices are reported by ${x}DeviceGet (same as default)
+- `0`: only sub-device 0 in the first device is reported
```

Comment:
there are no sub devices when FLAT_DEVICE_HIERACHY is  "1"

Reply:
thanks @MichalMrozek . I have expanded this section to make it clearer and added some tables showing what is being exposed. It would look something like this when rendered:

![image](https://github.com/oneapi-src/level-zero-spec/assets/8868111/8918068c-e628-406d-8498-db9c2535882b)